src/utils/useFirestore/index.ts

```diff
@@ -18,8 +18,8 @@ if (!firebase.apps.length) {
 const firestore = firebase.firestore()
 
 // Comment out the following to pull from the live firestore DB
-if (import.meta.env.NODE_ENV !== 'production') {
-  firestore.useEmulator('localhost', 8080)
-}
+// if (import.meta.env.NODE_ENV !== 'production') {
+//   firestore.useEmulator('localhost', 8080)
+// }
 
```

Comment:
@nobrayner you might know this better...?

Reply:
Uhhh... Why. This is definitely a bad idea - now when you are developing, running tests, etc., you will be hitting up the LIVE DB.

There is already some data seeded into the firestore emulator, so I really don't get why this has been removed.

Reply:
Ok thought so